result/yoda.js

```diff
@@ -0,0 +1,27 @@
+if ("red" === color) {
+    // ...
+}
+if (color === "red") {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if ("red" === color) {
+    // ...
+}
+
+if (true == flag) {
+    // ...
+}
+
+if (5 > count) {
+    // ...
+}
+
+if (-1 < str.indexOf(substr)) {
+    // ...
+}
+
+if (0 <= x && x < 1) {
```

Comment:
__[development][eslint]__
* Error - Expected literal to be on the right side of <=. (yoda)
* Error - 'x' is not defined. (no-undef)